src/app/app.component.html

```diff
@@ -40,7 +40,31 @@
             <ion-item routerDirection='root' routerLink="/security">
               <ion-icon slot="start" name="settings"></ion-icon>
               <ion-label>
-                Security Settings
+                Device Security
+              </ion-label>
+            </ion-item>
+            <ion-item routerDirection='root' routerLink="/security">
```

Comment:
We no longer need documentation in this.
I think we can just simply have TrustPage only and remove everything else

Reply:
ok, didn't get that we wanted to get rid of all the other pages. 👍